index.js

```diff
@@ -1,5 +1,13 @@
-const scan = require('./dynamodb/scan');
+const { DynamoDB } = require('@aws-sdk/client-dynamodb');
+const { DynamoDBDocument } = require('@aws-sdk/lib-dynamodb');
 
-module.exports = {
-  scan
+const getItem = require('./dynamodb/get-item');
+
+const client = new DynamoDB({});
+const documentClient = DynamoDBDocument.from(client);
```

Comment:
There might be a case where you want to send in `options` to this client. Not sure how we would do that with this structure 🤔

Reply:
Right, we should probably make a decision on that so we don't have to change the interface later. One option is to expose a function instead from index.js where you can pass in an options object (or not) and that function could return the other functions (but not the actual client object). If so, we should default to the same instance if options are blank or the same.